src/software/sensor_fusion/sensor_fusion.h

```diff
@@ -102,6 +102,7 @@ class SensorFusion
     std::optional<Ball> ball;
     Team friendly_team;
     Team enemy_team;
+    std::mutex game_state_mutex;
```

Comment:
shouldn't this be in threaded sensor fusion?

Reply:
Think it was a bad merge. Pulled from upstream and it's no longer there.

Reply:
jk that wasn't true, but it's fixed now (removed)